arch/arm/boot/dts/imx6ul-ts7250v3.dts

```diff
@@ -0,0 +1,1200 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR X11)
+/*
+ * Copyright 2019-2022 Technologic Systems, Inc. dba embeddedTS
+ */
+
+/dts-v1/;
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+#include "imx6ul.dtsi"
+
+/ {
+	model = "embeddedTS i.MX6UL TS-7250-V3";
+	compatible = "technologic,ts7250v3", "fsl,imx6ul";
+
+	aliases {
+		ethernet0 = &fec1;
+		ethernet1 = &fec2;
+		pc104bus = &pc104bus;
+		serial8 = &fpga_uart0;
+		serial9 = &fpga_uart1;
+		serial10 = &fpga_uart2;
+		serial11 = &fpga_uart3;
+		serial12 = &fpga_uart4;
+		serial13 = &fpga_uart5;
+		serial14 = &fpga_uart6;
+		serial15 = &fpga_uart7;
+		serial16 = &fpga_uart8;
+		spi4 = &opencores_spi0;
+		spi5 = &opencores_spi1;
+		gpio5 = &fpga_bank0;
+		gpio6 = &fpga_bank1;
+		gpio7 = &fpga_bank2;
+		gpio8 = &fpga_bank3;
+		gpio9 = &fpga_bank4;
+		gpio10 = &fpga_bank5;
+		gpio11 = &pc104gpio;
+	};
+
+	chosen {
+		stdout-path = &uart1;
+	};
+
+	memory {
+		device_type = "memory";
+		reg = <0x80000000 0x20000000>;
+	};
+
+	led-controller {
+		pinctrl-names = "default";
+		pinctrl-0 = <&pinctrl_cpu_leds>;
+		compatible = "gpio-leds";
+
+		led-0 {
+			color = <LED_COLOR_ID_GREEN>;
+			function = LED_FUNCTION_POWER;
+			gpios = <&gpio1 19 GPIO_ACTIVE_LOW>;
+			default-state = "on";
+		};
+
+		led-1 {
+			color = <LED_COLOR_ID_RED>;
+			function = LED_FUNCTION_INDICATOR;
+			gpios = <&gpio1 18 GPIO_ACTIVE_LOW>;
+			default-state = "off";
+		};
+	};
+
+	reserved-memory {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		ranges;
+
+		linux,cma {
+			compatible = "shared-dma-pool";
+			reusable;
+			size = <0x14000000>;
+			linux,cma-default;
+		};
+	};
+
+	regulators {
```

Comment:
Regulator nodes should be one level higher, the `regulator` sub node is not needed and can be detrimental. i.e. remove `regulators { ... } and move the actual regulator nodes up one level.